src/pages/designing/figma-kit.mdx

```diff
@@ -5,22 +5,162 @@ description: The new Figma design kit contains type styles, color tokens, and co
+</Caption>
+
+### Components
+
+Carbon for IBM.com components are available in the design kit.
+
+#### To bring components into your file:
+
+1. Have one or more of the Carbon for IBM.com libraries active.
+2. In the Layers panel, select `Layers → Assets`.
+3. Choose one of the Carbon for IBM.com themes.
+4. Select a component and drag it onto the canvas.
+
+![Using component variants](../../images/get-started/Designing_Components_Figma.gif)
+
+<Caption>
+  Components can be dragged from the Assets panel onto the canvas
+</Caption>
 
-We are currently working on creating separate kits for each theme so things may
-change but the alpha kit is great if you want to dive in and get started with
-Figma. Stay tuned for updates!
+#### Component variants
+
+Some components contain variants that allow you to change the properties of a component such as breakpoint, type, and interactive state. If a component contains variants, you will see them in the Design panel when you select a component. Before making updates directly to a component, check to see if the option is available as a variant.
+
+![Using component variants](../../images/get-started/Designing_Component_Variants_Figma.gif)
+
+<Caption>Component variants are accessed in the Design panel</Caption>
+
+#### Description and documentation
+
+Some components will contain additional guidance and links to documentation. When selecting a component, check the Design panel for a description and link to any relevant documentation.
+
+### Text styles
+
+Carbon for IBM.com text styles are available in each library.
+
+#### To apply text styles to text:
+
+1. Have one or more of the Figma libraries active.
+2. Select the text to change.
+3. In the `Design panel → Text`, click on `Style`.
+4. Select the text style to apply. Hover for additional usage guidance.
+
+Text styles within components are carefully considered. It’s important that you do not detach the symbol to change default styles.
+
+![Text styles in Figma](../../images/get-started/Designing_Text_Styles_Figma.jpg)
+
+<Caption>
+  In the Design panel, open the text styles overlay in the “Text” section to
+  view and apply text styles
+</Caption>
+
+### Color styles
+
+Carbon for IBM.com color styles are available in each library.
+
+#### To apply color styles to an element:
+
+1. Have one or more of the Figma libraries active.
+2. Select the text to change.
+3. In the `Design panel → Fill`, click on `Style`.
+4. Select the text style to apply. Hover for additional usage guidance.
+
+![Color styles in Figma](../../images/get-started/Designing_Color_Styles_Figma.jpg)
+
+<Caption>
+  In the Design panel, open the color styles overlay in the “Fill” section to
+  view and apply color styles
+</Caption>
+
+### Other Figma resources
+
+#### Bring in colors
+
+Additional color collections live in the IBM Design Systems libraries. In the Libraries overlay in Figma, select or search for “Color Styles - IBM Design Language”.
```

Comment:
It's actually not "IBM Design Systems libraries" but "IBM Design Language libraries". Design system = Carbon, which will have the same colors as us.

Also instead of "Additional color collections", maybe "For full IBM brand palette colors"